### ISSUE TITLE
Processes plugin: Provide actual disk I/O per process

### ISSUE THREAD

**Issue by PhilippWendler** [collectd/collectd#1225]
Under Linux, the processes plugin reads the values `rchar` and `wchar` from `/proc/<pid>/io` and exposes them as `ps_disk_octets`. These values include I/O that was satisfied by the page cache and even tty I/O.
There are also the values `read_bytes` and `write_bytes` which are an estimate of how much bytes were actually read from/written to the disk. For me, these values would be more interesting, thus I would like to propose that the processes plugin also exposes them.

Documentation: https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/Documentation/filesystems/proc.txt?id=v4.1#n1486

**Comment by bradjorgensen** [comment]
I'm using 5.4 and I've made several modifications to the source I use on my servers including what you mention here.  I'm planning on submitting most of them in git sometime hopefully before the end of the year.

One thing that will need to be decided, though, is the naming and types.  As you mention, the current metrics use the type `ps_disk_octets`, but it doesn't really represent disk traffic.  Because of that, I'm not sure if I should create a new type as I did in my local code or use a type instance.
